services/QuillLMS/app/workers/recalculate_staggered_release_locks_for_recently_completed_activities_worker.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class RecalculateStaggeredReleaseLocksForRecentlyCompletedActivitiesWorker
+  include Sidekiq::Worker
+
+  def perform
+    recently_completed_activity_sessions.each do |activity_session|
+      SaveUserPackSequenceItemsWorker.perform_async(activity_session.user_id, activity_session.classroom_id)
+    end
+  end
+
+  private def recently_completed_activity_sessions
+    ActivitySession
+      .joins(:classroom_unit)
+      .where(completed_at: 2.hours.ago..1.hour.ago)
```

Comment:
Given that the cron timing is a little fuzzy, should we broaden this window slightly in case we run five minutes off between calls and end up with a tiny gap that's not covered?

Reply:
Oh.  That's good point.  Will change this to `125.minutes.ago...55.minutes.ago`